### ISSUE TITLE
"Resolve error: TypeError: Cannot read property 'lastIndexOf' of null" after updating to 2.21.2

### ISSUE THREAD

**Issue by sceee** [benmosher/eslint-plugin-import#1826]
After updating from 2.20.2 to 2.21.2 in a typescript project, I am facing the following error:
```
1:1  error    Resolve error: TypeError: Cannot read property 'lastIndexOf' of null
    at Object.exports.resolve (C:\proj\node_modules\eslint-import-resolver-webpack\index.js:30:26)
    at v2 (C:\proj\node_modules\eslint-module-utils\resolve.js:117:23)
    at withResolver (C:\proj\node_modules\eslint-module-utils\resolve.js:122:16)
    at fullResolve (C:\proj\node_modules\eslint-module-utils\resolve.js:139:22)
    at relative (C:\proj\node_modules\eslint-module-utils\resolve.js:84:10)
    at resolve (C:\proj\node_modules\eslint-module-utils\resolve.js:220:12)
    at resolveImportType (C:\proj\node_modules\eslint-plugin-import\lib\core\importType.js:149:65)
    at computeRank (C:\proj\node_modules\eslint-plugin-import\lib\rules\order.js:315:44)
    at registerNode (C:\proj\node_modules\eslint-plugin-import\lib\rules\order.js:331:16)
    at handleImports (C:\proj\node_modules\eslint-plugin-import\lib\rules\order.js:600:9)
    at C:\proj\node_modules\eslint\lib\linter\safe-emitter.js:45:58
    at Array.forEach (<anonymous>)
    at Object.emit (C:\proj\node_modules\eslint\lib\linter\safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (C:\proj\node_modules\eslint\lib\linter\node-event-generator.js:254:26)
    at NodeEventGenerator.applySelectors (C:\proj\node_modules\eslint\lib\linter\node-event-generator.js:283:22)
    at NodeEventGenerator.enterNode (C:\proj\node_modules\eslint\lib\linter\node-event-generator.js:297:14)  import/order
    7:1  error    `./other` import should occur after import of `null`
```

The imports in the file which seems to be causing this error look like this:
```
import firebase, { User } from 'firebase/app'
import 'firebase/analytics'
import 'firebase/auth'
import 'firebase/firestore'
import 'firebase/storage'
import 'firebase/remote-config'
import Other from './other'
...
```

`other` has no imports, just a default export.

**Comment by ljharb** [comment]
I've just published an updated eslint-import-resolver-webpack; any chance that fixes it? Your test case passes on master, altho I can't be certain without your eslint config for the `order` rule.

**Comment by sceee** [comment]
@ljharb I tried it with an explicitly installed `eslint-import-resolver-webpack@0.12.2` to lock it to v0.12.2 but unfortunately, the same error still occurs.
Actually, I don't have an explicit config for the `order` rule, I use
```
extends: [
    'plugin:vue/essential',
    '@vue/airbnb',
    '@vue/typescript/recommended',
    '@vue/prettier',
    'prettier/@typescript-eslint',
  ],
```

The effective config for `import/order` seems to be
```    
"import/order": [
      "error",
      {
        "groups": [
          [
            "builtin",
            "external",
            "internal"
          ]
        ]
      }
    ],
```

**Comment by ljharb** [comment]
Is the stack trace still identical? Same line number out of `eslint-import-resolver-webpack`?

**Comment by sceee** [comment]
@ljharb sorry, I did oversee this part - the stack looked the same at first but I oversaw the first line differs slightly. Sorry for that.

The error now comes from
```
    1:1  error    Resolve error: TypeError: Cannot read property 'lastIndexOf' of null
    at Object.exports.resolve (C:\proj\node_modules\@vue\eslint-config-airbnb\node_modules\eslint-import-resolver-webpack\index.js:30:26)
```
because `eslint-import-resolver-webpack` is still a `0.x` and `@vue/eslint-config-airbnb` requires `"eslint-import-resolver-webpack": "^0.11.1",`.

So updating regularly using npm is not possible as far as I know (as it will always install the `0.11.1` for `@vue/eslint-config-airbnb` even if I install `0.12.2` itself to the project because of the `^0.x` semver range that is treated as major).

But for testing I copied the `0.12.2` directory to the `node_modules` of `@vue/eslint-config-airbnb` and then the compile worked again.

So it seems to be fixed in `0.12.2` but that does not help me immediately as `@vue/eslint-config-airbnb` needs to bump dependencies. I will create an issue over there.
